_includes/research-platform/projects-collapsable.html

```diff
@@ -0,0 +1,35 @@
+<section id="projects" class="container content-section">
+    <div class="row">
+        <div class="col-lg-12">
+            <p>
+                We aim to connect students and research leaders in ML/AI 
+                by helping the students discover projects suited to their interests
+                and research labs showcase their work and available opportunities
+                <a class="btn btn-default" href="{{ "./about" | relative_url }}">Learn more about the platform</a>
+            </p>
+            <p>
+                Are you a student and don't know how to start your adventure with ML/AI research?
+                <a class="btn btn-default" href="{{ './tips-for-students' | relative_url }}">See our tips for young researchers</a>
+            </p>
+
+            <div class="search-bar input-group input-group-lg">
```

Comment:
It looks cool, but I would find filtering options more useful than a search bar.

Reply:
In the following versions, we can have filters too! This search bar doesn't work, so it's only here to sell the idea that it will be a searchable database :)